src/System.Private.CoreLib/src/System/Threading/Overlapped.cs

```diff
@@ -242,23 +181,20 @@ internal IntPtr UserHandle
 
     public class Overlapped
     {
-        private OverlappedData m_overlappedData;
-        private static PinnableBufferCache s_overlappedDataCache = new PinnableBufferCache("System.Threading.OverlappedData", () => new OverlappedData());
+        private OverlappedData _overlappedData;
```

Comment:
The split between Overlapped and OvelappedData is not necessary either anymore. I may be able to eliminate the split as well.